publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java

```diff
@@ -259,12 +261,12 @@ void shouldGivePermissionToOperateOnPublicationWhenEditor() throws JsonProcessin
     }
 
     private static Function<AccessRight, String> getCognitoGroup(URI institutionId) {
-        return accessRight -> accessRight.name() + AT + institutionId.toString();
+        return accessRight -> accessRight.toPersistedString() + AT + institutionId.toString();
     }
 
     private List<AccessRight> getCuratorWithPublishDegreeAccessRight() {
         var curatorAccessRight = getCuratorAccessRights();
-        curatorAccessRight.add(AccessRight.PUBLISH_DEGREE);
+        curatorAccessRight.add(MANAGE_PUBLISHING_REQUESTS);
```

Comment:
MANAGE_PUBLISHING_REQUESTS er gamle PUBLISH_DEGREE?

Reply:
Think this should be mapped to `MANAGE_DEGREE`?